grudge/trace_pair.py

```diff
@@ -214,11 +215,19 @@ def bv_trace_pair(
 
 # {{{ Interior trace pairs
 
-def _interior_trace_pair(dcoll: DiscretizationCollection, vec) -> TracePair:
+def interior_trace_pair(dcoll: DiscretizationCollection, vec) -> TracePair:
```

Comment:
I'm not loving the naming now. `interior_trace_pair` is local-only, vs `interior_trace_pairs` is also remote? It's not obvious to me. Maybe make the un-deprecated `interior_trace_pair` available as `local_interior_trace_pair`?

Reply:
Good call, the name was bothering me as well :smile: I'll update soon!

Reply:
Done (included doc updates): https://github.com/inducer/grudge/pull/186/commits/97078b6a8fb4bd2121869c17342186ae47c8f5aa